www/index.js

```diff
@@ -1717,6 +1724,9 @@ function update_session() {
     if (session.status == "google_oauth_registering") {
       push_menu_page(menu_signup_with_google_page);
     }
+    if (session.status == "lichess_oauth_registering") {
```

Comment:
please also update the other place where "google_oauth_registering" is used

Reply:
Done